cmake/test_config/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Why this?

Reply:
I suppose it isn't necessary, just avoids a dirty repo if the test program is built in source.

Reply:
In theory this would only be built on CI, but it can also be tested locally